js/data/locale/af/regionnames.json

```diff
@@ -1,32 +1,32 @@
 {
     "AD": [
         {
-            "code": "02",
-            "name": "Canillo"
+            "name": "Canillo",
```

Comment:
Why is this one not sorted properly? "code" should come before "name" ...

Reply:
yeah...I re-updated the tool code and generated `regionnames.json` files again to have the correct sorted data. Please take a look them again